lib/signaling/v2/room.js

```diff
@@ -758,8 +758,10 @@ function handleTransportEvents(roomV2, transport) {
  * @param {Number} intervalMs
  */
 function periodicallyPublishStats(roomV2, transport, intervalMs) {
+  let evenPublishCount = true;
   const interval = setInterval(() => {
     roomV2.getStats().then(stats => {
+      evenPublishCount = !evenPublishCount;
```

Comment:
Minor - It may be better to publish ice candidate pair stats on the first period.